e2e-tests/data-at-rest-encryption/run

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+create_namespace $namespace
+deploy_operator
+deploy_vault
+deploy_minio
+
+apply_s3_storage_secrets
+
+desc 'create PSMDB cluster'
+
+cluster="some-name"
+kubectl_bin apply -f "$conf_dir/secrets.yml" -f "$conf_dir/client.yml"
+
+apply_cluster "$test_dir/conf/$cluster.yml"
+desc 'check if all 3 Pods started'
+wait_for_running $cluster-rs0 3
+wait_for_running $cluster-cfg 3 "false"
+wait_for_running $cluster-mongos 3
+
+compare_kubectl statefulset/$cluster-rs0
+compare_kubectl statefulset/$cluster-cfg
```

Comment:
We don't seem to be doing any backup/restores, but we have backup storages and backup files defined.
I was thinking that we should maybe have just one backup/restore (not for all storages).

Reply:
I can return backup/restore code just to test that we can do it) Any problem.